Credify/Credify/serviceX.swift

```diff
@@ -116,45 +117,93 @@ public struct serviceX {
         ///   - userProfile: User object
         ///   - pushClaimTokensTask: A task that calls your push claim token API. This SDK needs to receive success status of this task.
         ///   - completionHandler: Completion handler. You can get notified about the result of the offer redemption flow.
+        @available(*, deprecated, message: "Using presentByOfferCodeModally method instead")
```

Comment:
Please update the function name accordingly

Reply:
@ngo275 I updated it